### DISCUSSION TITLE
Qfield search parameters

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#3866] [Q&A]
Hi, 

Why is there the possibility to search adress in Finland with the prefixe fia ? Why is it only for finland ?

Thanks

**Comment by lucienicolier** [comment]
Hi,

Someone in Finland paid the feature. 
If you are interested, you can [sponsor ](https://docs.qfield.org/get-started/sponsor/#sponsor-qfield) the feature for another country.
